### PR TITLE
feat(confidential-ledger): update append to handle a passed guid as /…

### LINKED ISSUE #113: Append by given guid

### PR DESCRIPTION
**Pull request by FergusKidd**
…append/{guid here}

fixes #113

### CODE REVIEW

File: confidential-ledger-demo/server/app.py [hunk 1]
```diff
@@ -74,12 +73,18 @@ async def append(request):
         return JSONResponse({"error": "Body is not a valid json"})
 
     try:
-        guid, returnData = cl.append_cl(bodyData)
+        returnData = cl.append_cl(bodyData, newEntryId)
```
Comment (Sealjay-clj) [inline confidential-ledger-demo/server/app.py:76]: please use snake_case

File: confidential-ledger-demo/server/app.py [hunk 2]
```diff
@@ -62,8 +62,7 @@ async def readlicense(request):
 
 
 async def append(request):
-    # TODO: What do we do if we want to add data to an existing guid?
-    newEntryId = str(uuid.uuid4())
+    newEntryId = request.path_params["guid"]
 
     bodyData = await request.body()
```
Comment (Sealjay-clj) [inline confidential-ledger-demo/server/app.py:67]: please use snake case

File: confidential-ledger-demo/server/app.py [hunk 3]
```diff
@@ -62,8 +62,7 @@ async def readlicense(request):
 
 
 async def append(request):
-    # TODO: What do we do if we want to add data to an existing guid?
-    newEntryId = str(uuid.uuid4())
+    newEntryId = request.path_params["guid"]
```
Comment (Sealjay-clj) [inline confidential-ledger-demo/server/app.py:65]: please use snake case

**Review by Sealjay-clj — COMMENTED** [review]
some snake case issues @FergusKidd - otherwise free to merge when those are fixed

File: confidential-ledger-demo/server/app.py (hunk 3 above)
Comment (FergusKidd) [reply confidential-ledger-demo/server/app.py:65]: sorry

File: confidential-ledger-demo/server/app.py (hunk 2 above)
Comment (FergusKidd) [reply confidential-ledger-demo/server/app.py:67]: sorry

File: confidential-ledger-demo/server/app.py (hunk 1 above)
Comment (FergusKidd) [reply confidential-ledger-demo/server/app.py:76]: sorry

**Pull request merged**